StyleCop.Analyzers/StyleCop.Analyzers.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -196,7 +196,7 @@ private static void VerifyDiagnosticLocation(DiagnosticAnalyzer analyzer, Diagno
             var actualLinePosition = actualSpan.StartLinePosition;
 
             // Only check line position if there is an actual line in the real diagnostic
-            if (actualLinePosition.Line > 0)
+            if (actualLinePosition.Line >= 0)
```

Comment:
The bug is actually that `actualLinePosition` is used here. The original intent was for the user to be able to specify in the expected results that the line doesn't matter. The condition should be this:

```csharp
if (expected.Line > 0)
```